patternfly/patternfly-elements
# fix: added resize observer for lining up progress bar

## <Pull request description>

The progress bar line is dynamically calculated based on the size of the items.  Because of this, we need to add a resize event so let the progres-steps element know to recalculate the bar.


### Related issues

- #1710 


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1763--patternfly-elements.netlify.app/elements/pfe-progress-steps/demo/) 


### What has changed and why



- Resize event listener added
- Removed `min-width: fit-contents:` so that the `--pfe-progress-steps__item--size` property works as expected.


### Testing instructions



- [ ] *Verify that resize observer looks good*
  1. Go here https://deploy-preview-1763--patternfly-elements.netlify.app/elements/pfe-progress-steps/demo/
  2. Load the website on desktop size
  3. Shrink the browser size down to mobile size
  4. Verify that all of the progress bars span the length of the first to last item
 
- [ ] *Verify that all items share consistent widths*
  1. Go here https://deploy-preview-1763--patternfly-elements.netlify.app/elements/pfe-progress-steps/demo/
  2. Open your console and past in the "Demo Snippet" code below.
  3. You should see variable size text in the first item.
  4. The middle item should be centered on the page which indicates that all items have the same widths
  4. Verify that the progress bar spans the length of the first to last item


#### Demo Snippet

```
let progressStep = document.querySelector('pfe-progress-steps');
let item = progressStep.querySelector('[slot=title]').innerText = "aoisdnf sfosdf asdf oasf";
progressStep.style.setProperty('--pfe-progress-steps__item--size', '300px');
progressStep._build();
```

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
